### PR TITLE
ncdu, popd, pushd, w: add Danish translation

### PR DESCRIPTION
**Pull request by drasbeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by drasbeck** [comment]
Pushed the fix. Is this PR missing something?

**Comment by drasbeck** [comment]
@kbdharun, sorry to drag you in here, but my PR seems to be stuck, any idea why?

File: pages.da/common/w.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# w
+
+> Vis hvem der er logget ind og hvad de laver.
+> Print bruger login, TTY, fjernforbundet vært, login-tidspunkt, inaktiv tid, nuværende proces.
+> Mere information: <https://manned.org/w>
```
Comment (kbdharun) [inline pages.da/common/w.md:5]:
```suggestion
> Mere information: <https://manned.org/w>.
```

File: pages.da/common/ncdu.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ncdu
+
+> Diskforbrugsanalyse i ncurses interface.
```
Comment (kbdharun) [inline pages.da/common/ncdu.md:3]: I checked this line with Google translate. And it wasn't displaying properly for me. Can this be used?

```suggestion
> Diskbrugsanalysator med en ncurses-grænseflade.
```

**Comment by kbdharun** [comment]
> @kbdharun, sorry to drag you in here, but my PR seems to be stuck, any idea why?

I think the missing period in 1 file might have caused the issue. With my 2 suggestions above, addressed this PR is GTG.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**